napari/settings/_tests/test_migrations.py

```diff
@@ -0,0 +1,66 @@
+from importlib.metadata import distribution
```

Comment:
@nclack @Carreau, curious if you think these tests are going in the right direction